src/magicbox/SuggestionsManager.ts

```diff
@@ -221,6 +224,36 @@ export class SuggestionsManager {
     });
   }
 
+  private get querySuggestPreviewComponent() {
+    const querySuggestPreviewElement: HTMLElement = $$(this.root).find(`.${Component.computeCssClassNameForType('QuerySuggestPreview')}`);
+    if (!querySuggestPreviewElement) {
+      return;
+    }
+    const querySuggestPreview = <QuerySuggestPreview>Component.get(querySuggestPreviewElement);
+    if (!(querySuggestPreview.options.numberOfPreviewResults > 0)) {
```

Comment:
the minimum for that option (numberOfPreviewResults) is 1, so I don't think you have to manage this case.
In any case you could have used `querySuggestPreview.options.numberOfPreviewResults <= 0` ?

Reply:
You are right.
This is probably from before I had a default value, min and max so I checked if I had any results to display

Will fix in my next commit